apps/google-analytics-4/.gitignore

```diff
@@ -18,6 +18,9 @@
 .env.test.local
 .env.production.local
 
+# dotenv
+/**/*/.env
```

Comment:
I wonder if we need this? Maybe instead we should suggest instrumenting the env with `.env.local` in our README so that we don't have to exclude this file. (I've seen some projects where keeping an .env with defaults in repo is a standard, with the expectation that overrides happen wih the .local pattern, e.g. what's above in the gitignore file a few linews)

Reply:
the frontend/.env.development file is _not_ gitignored currently and is holding non-sensitive configuration data.